docs/classical_control.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<p>We can link to the Textbook Algorithms in Cirq workshop for teleportation and this <a href="https://www.nature.com/articles/s41586-021-03588-y" target="_blank">paper</a> (linked from quantumai.google/research) for teleportation and error correction, respectively, unless you have other preferred links.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5403/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done.